src/firefly/js/ui/ExampleDialog.jsx

```diff
@@ -179,7 +178,7 @@ class FieldGroupTest extends Component {
 
     constructor(props)  {
         super(props);
-        FieldGroupUtils.initFieldGroup('DEMO_FORM',exDialogReducer);
+        dispatchInitFieldGroup('DEMO_FORM',false,null,exDialogReducer);
```

Comment:
If fftools.js lines 102-158 you can see my experimenting with taking out the connector. It works fine and does eliminate one of the warning messages.